### ISSUE TITLE
Improve new remote dialog

### ISSUE THREAD

**Issue by jecisc** [pharo-vcs/iceberg#1088]
With the plugins we could improve the new remote dialog to fetch the forks of the project from the hosts (for github, bitbucket, gitlab) and display the list of possible remotes to the user.

**Comment by guillep** [comment]
Yes, we could actually just use the same UI than the "new repository" window
